views/prisonerProfile/macros/prisonerSummarySection.njk

```diff
@@ -17,7 +17,13 @@
 
   {% for detail in array %}
 
-    {% if detail.html %}
+    {% if detail.ldd %}
```

Comment:
Yes - like this, but...

We ought to give it a generic name instead of `ldd` (and `lddList`) - maybe just `detail.values`?

Can be fixed later though.